include/udx.h

```diff
@@ -82,6 +82,13 @@ typedef struct udx_socket_send_s udx_socket_send_t;
 typedef struct udx_stream_write_s udx_stream_write_t;
 typedef struct udx_stream_send_s udx_stream_send_t;
 
+/* todo: should this be merged into the stream_write_t object? */
```

Comment:
If there's always a 1:1 correspondence between them, I'd merge them.

Reply:
for now the stream_write call only accepts a single buffer, but the plan is still to allow an multiple bufs to be passed to a single write, so it makes sense to keep the separate object. I removed the todo.